### ISSUE TITLE
[Hunter Pet] Vargul Blighthound pet does significantly less damage than other wolves

### ISSUE THREAD

**Issue by arestavo** [dalaranwow/dalaran-wow#5800]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
When tamed, and assuming same hunter/pet spec and levels, the Vargul Blighthound from the zone Zul'Drak does SIGNIFICANTLY less damage than other wolves. 201.7 DPS vs 336.1, same exact spec, level, and buffs.
**Expected behaviour**: Tell us what should happen instead
All wolves should do the same damage.
**Steps to reproduce the problem**:

1. Tame a Vargul Blighthound from Zul'Drak.
2. Level it to 80, or compare it to a same level wolf.
3. Notice that it does a lot less damage comparatively.

**Include proofs for this behaviour**
![WoWScrnShot_072819_194429](https://user-images.githubusercontent.com/50332138/62016837-8b6b5a00-b171-11e9-96a9-38327c2e6998.jpg)
![WoWScrnShot_072819_194929](https://user-images.githubusercontent.com/50332138/62016840-8efee100-b171-11e9-944a-0c65ae9d7576.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by arestavo** [comment]
I figured it out - neither Dalaran's WoW client, or Warmane's WoW client (with Dalaran's server info) properly displays pet happiness.

Without being able to see the pet's actual happiness (which shows as a green, yellow, or red face), there's no way to tell whether or not the pet is happy and doing happy levels of damage.
